### PR TITLE
Vsphere docs

### PR DESCRIPTION
**Pull request by joelddiaz**

### CODE REVIEW

File: docs/hiveutil.md [hunk 1]
```diff
@@ -55,6 +55,25 @@ bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=gcp myclu
...
+ovirt_fqdn: ""
+ovirt_pem_url: ""
+ovirt_username: admin@internal
+ovirt_password: secretpassword
+ovirt_ca_bundle: |-
+  -----BEGIN CERTIFICATE-----
+  CA CERT DATA
+  -----END CERTIFICATE-----
+```
+
```
Comment (akhil-rane) [inline docs/hiveutil.md:72]: I think hiveutil command to create cluster is missing here

File: docs/hiveutil.md [hunk 2]
```diff
@@ -55,6 +55,25 @@ bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=gcp myclu
...
+#### Create Cluster on vSphere
+
+Set credentials/connection information in the following environment variables. `GOVC_USERNAME` should hold the vSphere username, `GOVC_PASSWORD` should be set to the vSphere user's password. If the vCenter instance is using self-signed certificates or is otherwise untrusted by the system being used to connect to vCenter, `GOVC_TLS_CA_CERTS` should be set to the path of a file containing the CA certificate for the vCenter instance. 
+
```
Comment (akhil-rane) [inline docs/hiveutil.md:76]: hiveutil command is missing

File: docs/using-hive.md [hunk 3]
```diff
@@ -160,6 +160,22 @@ metadata:
 type: Opaque
 ```
 
+#### vSphere
+Create a `secret` containing your vSphere credentials information:
+
+```yaml
+apiVersion: v1
+data:
+data:
```
Comment (staebler) [inline docs/using-hive.md:169]: Duplicate of previous line.

File: docs/using-hive.md [hunk 4]
```diff
@@ -160,6 +160,22 @@ metadata:
 type: Opaque
 ```
 
+#### vSphere
+Create a `secret` containing your vSphere credentials information:
+
+```yaml
+apiVersion: v1
+data:
```
Comment (staebler) [inline docs/using-hive.md:168]: Should this be `stringData`?

File: docs/using-hive.md [hunk 5]
```diff
@@ -276,6 +292,30 @@ platform:
     ovirt_storage_domain_id: 00000000-storage-domain-uuid
 ```
 
+For vSphere, replace the contents of `compute.platform` and `controlPlane.platform` with emtpy brackets(`{}`).
+```yaml
+controlPlane:
+  platform: {}
```
Comment (staebler) [inline docs/using-hive.md:298]: Can't this be omitted instead?

File: docs/using-hive.md [hunk 6]
```diff
@@ -340,6 +380,22 @@ ovirt:
   storage_domain_id: 00000000-storage-domain-uuid
 ```
 
+
+For vSphere, replace the contents of `spec.platform` with:
+```yaml
+vsphere:                                                                    
+  certificatesSecretRef:         
+    name: jdiaz-vsphere-certs
```
Comment (staebler) [inline docs/using-hive.md:388]:
```suggestion
    name: mycluster-vsphere-certs
```

File: docs/using-hive.md [hunk 7]
```diff
@@ -340,6 +380,22 @@ ovirt:
...
+  cluster: devel
+  credentialsSecretRef:                                                     
+    name: jdiaz-vsphere-creds
```
Comment (staebler) [inline docs/using-hive.md:391]:
```suggestion
    name: mycluster-vsphere-creds                                                                                                                          
```

File: docs/using-hive.md [hunk 8]
```diff
@@ -395,6 +451,16 @@ ovirt:
     sizeGB: 120
 ```
 
+For vSphere, replace the contents of `spec.platform` with the settings you want for the instnaces:
```
Comment (staebler) [inline docs/using-hive.md:454]:
```suggestion
For vSphere, replace the contents of `spec.platform` with the settings you want for the instances:
```

**Comment by joelddiaz** [comment]
/unhold
ovirt docs merged #1126

**Review by staebler — APPROVED** [review]
/lgtm

**Pull request merged**